source/jormungandr/jormungandr/stat_manager.py

```diff
@@ -209,7 +209,7 @@ def fill_request(self, stat_request, call_result):
             stat_request.client = request.headers.getlist("X-Forwarded-For")[0]
         stat_request.path = request.path
 
-        stat_request.response_size = sys.getsizeof(call_result[0])
+        stat_request.response_size = 4242
```

Comment:
This line should be removed or commented. 

The intention of using `sys.getsizeof()` is to get the size of response in terms of 'length', but in fact this function returns the whole size of a python object in memory, the use may be wrong  and secondly this function is not yet implemented in pypy...